iceoryx2/src/service/static_config/message_type_details.rs

```diff
@@ -120,3 +122,124 @@ impl MessageTypeDetails {
             && self.payload.alignment <= rhs.payload.alignment
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use iceoryx2_bb_testing::assert_that;
+
+    #[test]
+    fn test_from() {
+        struct MyPayload {
```

Comment:
Must be annotated with `#[repr(C)]` - in one binary it would not make any difference but just for completeness.